maintenance/ApplyRules.php

```diff
@@ -5,6 +5,8 @@
 namespace ProfessionalWiki\Rules\Maintenance;
 
 use Maintenance;
```

Comment:
You can use `MediaWiki\Maintenance\Maintenance` instead so that i works on 1.44+